gazebo_ros/src/gazebo_ros_api_plugin.cpp

```diff
@@ -249,9 +248,6 @@ void GazeboRosApiPlugin::advertiseServices()
     return;
   }
 
-  // publish clock for simulated ros time
-  pub_clock_ = nh_->advertise<rosgraph_msgs::Clock>("/clock",10);
-
```

Comment:
And now this code is duplicate because we already do this in the main `loadGazeboRosApiPlugin`